gclient/tests/perf.rs

```diff
@@ -0,0 +1,265 @@
+        ("demo_sum", 42.encode()),
+        ("demo_vec", 65535.encode()),
+        (
+            "demo_async_tester",
+            demo_async_tester::Kind::SendCommit.encode(),
+        ),
+        ("demo_btree", demo_btree::Request::Insert(42, 84).encode()),
+        (
+            "demo_calc_hash_in_one_block",
+            demo_calc_hash_in_one_block::Package {
+                expected: 6400,
+                package: demo_calc_hash::Package {
+                    result: [0u8; 32],
+                    counter: 0,
+                },
+            }
+            .encode(),
+        ),
+        (
+            "demo_distributor",
+            demo_distributor::Request::Report.encode(),
+        ),
+        ("demo_exit_handle", vec![]),
+        (
+            "demo_exit_handle_sender",
+            demo_exit_handle_sender::Input::SendMessage {
+                destination: [0u8; 32].into(),
+                payload: vec![],
+                value: 0,
+            }
+            .encode(),
+        ),
+        ("demo_gas_burned", vec![]),
+        (
+            "demo_gasless_wasting",
+            demo_gasless_wasting::InputArgs {
+                prog_to_wait: [0u8; 32].into(),
+                prog_to_waste: [0u8; 32].into(),
+            }
+            .encode(),
+        ),
+        (
+            "demo_meta",
+            demo_meta::MessageIn {
+                id: demo_meta::Id {
+                    decimal: 1,
+                    hex: vec![1u8],
+                },
+            }
+            .encode(),
+        ),
+        ("demo_mul_by_const", 42_u64.encode()),
+        ("demo_ncompose", vec![1, 2, 3, 4]),
+        (
+            "demo_new_meta",
+            demo_meta_io::MessageIn {
+                id: demo_meta_io::Id {
+                    decimal: 1,
+                    hex: vec![1u8],
+                },
+            }
+            .encode(),
+        ),
+        ("demo_node", demo_node::Request::Add(42).encode()),
+        (
+            "demo_program_factory",
+            demo_program_factory::CreateProgram::Default.encode(),
+        ),
+        ("demo_proxy", vec![0; 32_768]),
+        ("demo_proxy_relay", vec![0; 32_768]),
+        ("demo_proxy_with_gas", 1_000_000_000_u64.encode()),
+    ];
+
+    let mut block_gas_limit = api.block_gas_limit()?;
+    let mut messages = Vec::with_capacity(handle_payloads.len());
+    for (prog_name, payload) in handle_payloads {
+        let prog_id = progs[prog_name];
+        let gas_info = api
+            .calculate_handle_gas(None, prog_id, payload.clone(), 0, true)
+            .await?;
+        block_gas_limit = block_gas_limit.saturating_sub(gas_info.burned);
+        if block_gas_limit == 0 {
+            break;
+        }
+        messages.push((prog_id, payload, gas_info.min_limit, 0));
+    }
+
+    println!("Message count: {}", messages.len());
+
+    assert!(block_gas_limit == 0); // TODO: Uncomment after implementation
```

Comment:
Sorry comment is confusing a bit. And imho it is better to not place comments aside )

Reply:
Ah, forgot to remove it after uncommenting the line itself 🙈 
Thanks!